src/main/resources/templates/start.html

```diff
@@ -1,35 +1,25 @@
  <!DOCTYPE html>
-<html lang="en">
+ <html lang="en" xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
     <title>BulkBuy</title>
 
-    <script type="text/javascript">
-        document.getElementById("SearchOrder").onclick = function () {
-            location.href = "orderLookup.html";
-        };
-    </script>
-
 </head>
 <body>
-BulkBuy Dashboard
-
-
-<button id="PlaceOrder" class="float-left submit-button" >Place New Order</button>
-
-<script type="text/javascript">
-    document.getElementById("PlaceOrder").onclick = function () {
-        location.href = "newOrder.html";
-    };
-</script>
-
-<button id="SearchOrder" class="float-left submit-button" >Lookup Order</button>
+    <div style="display: inline-block">
+        BulkBuy Dashboard
+    </div>
+    <div style="display: inline-block">
+        <form action="#" class="float-left submit-button" th:action="@{/newOrder}" method="get" >
+            <button name="submit" type="submit" value="0">Place New Order</button>
+        </form>
+    </div>
+    <div style="display: inline-block">
+        <form action="#" class="float-left submit-button" th:action="@{/orderLookup}" method="get" >
+            <button name="submit" type="submit" value="0">Lookup Order</button>
+        </form>
+    </div>
```

Comment:
Replaced JavaScript event listener with form submit buttons using Thymeleaf to access the endpoints desired.
This will help standardize the code by using Thymeleaf to help create organizational efficiency.
https://www.capitalone.com/tech/software-engineering/10-microservices-best-practices/